src/Directory.Build.props

```diff
@@ -30,6 +30,19 @@
     <Compile Include="$(MSBuildThisFileDirectory)GlobalSuppressions.cs" Link="GlobalSuppressions.src.cs" />
 
     <!-- reference assemblies let us target .NET Framework without the SDK (for example, on non-Windows) -->
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3" PrivateAssets="all" />
```

Comment:
Opportunistic bump, as dependabot is handling this file. See #1312

Reply:
I want to at least test https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1314 before merging this PR.

Reply:
See https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1315